app/jobs/send_vacancy_application_change_job.rb

```diff
@@ -0,0 +1,17 @@
+class SendVacancyApplicationChangeJob < ApplicationJob
+  queue_as :default
+
+  def perform
+    Vacancy
+      .includes(:publisher)
+      .where(
+        receive_applications: :email,
+        created_at: 18.months.ago...,
+      )
+      .select(:contact_email).distinct
+      .select("*")
+      .find_each do |vacancy|
+        Publishers::VacancyChangeMailer.notify(vacancy:).deliver_later
```

Comment:
might be clearer for VacancyChangeMailer to take the publisher rather than the whole Vacancy. Do we not want distinct publishers rather than distinct contact emails?